d/d3-ease/d3-ease_rhel_8.4.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash -e
+# -----------------------------------------------------------------------------
+#
+# Package       : d3-ease
+# Version       : v1.0.7
+# Source repo   : https://github.com/d3/d3-ease
+# Tested on	: UBI 8.5
+# Language      : Node
+# Travis-Check  : True
+# Script License: Apache License, Version 2 or later
+# Maintainer    : Vathsala . <vaths367@in.ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+#Exit immediately if a command exits with a non-zero status.
+set -e
```

Comment:
not needed, you already have the same thing on Line #1 of the script

Reply:
Updated the script with required changes